CacheTagsBundle/Annotation/CacheTag/AbstractCacheTag.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace lbarulski\CacheTagsBundle\Annotation\CacheTag;
+
+use Doctrine\Common\Annotations\Annotation\Required;
+use Doctrine\Common\Annotations\Annotation\Target;
+use lbarulski\CacheTagsBundle\Tag\TagInterface;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\ConfigurationInterface;
+
+/**
+ * @Annotation
+ * @Target("METHOD")
+ */
+abstract class AbstractCacheTag implements TagInterface, ConfigurationInterface
```

Comment:
Annotation class shouldn't implement `TagInterface` ... it's the interface to be implemented on models instead